### PR TITLE
Idr0025

### PR DESCRIPTION
**Pull request by eleanorwilliams**
This is ready for testing on test34 along with Simone's plates.tsv files in https://github.com/IDR/idr-metadata/pull/211

### CODE REVIEW

**Comment by eleanorwilliams** [comment]
@aleksandra-tarkowska - could you check over the bulk-map config file to check that I have added mapr/antibody correctly.

**Comment by joshmoore** [comment]
And we'll need an ansible commit to add the menu. cc: @manics

File: idr0025-stadler-proteinatlas/screenA/idr0025-screenA-bulkmap-config.yml [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+      - name: Antibody Identifier
+        include: yes
+        omitempty: no
+      - name: Antibody Identifier
+        clientname: Antibody Identifier URL
+        clientvalue: http://www.proteinatlas.org/search/{{ value|urlencode }}
+        include: yes
+
+  - group:
+      namespace: openmicroscopy.org/mapr/antibody/supplemntary
```
Comment (aleksandra-tarkowska) [inline idr0025-stadler-proteinatlas/screenA/idr0025-screenA-bulkmap-config.yml]: supplementary

File: idr0025-stadler-proteinatlas/screenA/idr0025-screenA-bulkmap-config.yml [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+    - namespace: openmicroscopy.org/mapr/organism
+      keys:
+      - Organism
+    - namespace: openmicroscopy.org/mapr/sirna
+      keys:
+      - siRNA Identifier
+      - siRNA Pool Identifier
+    - namespace: openmicroscopy.org/mapr/antibody
+      keys:
+      - Antibody Identifier
```
Comment (aleksandra-tarkowska) [inline idr0025-stadler-proteinatlas/screenA/idr0025-screenA-bulkmap-config.yml]: https://github.com/IDR/deployment/pull/12 updated to match Antibody Identifier

**Comment by joshmoore** [comment]
Note: stealing `bulk.yml` from here in order to import on demo34, will merge when the annotations are ready.

**Comment by joshmoore** [comment]
Tested on test34. Merging for demo34.

**Pull request merged**